Parsers/ASimAuthentication/Parsers/ASimAuthenticationSalesforceSC.yaml

```diff
@@ -0,0 +1,342 @@
+        "X", "Admin"
+    ];
+        union isfuzzy=true
+        SalesforceSchema,
+        SalesforceServiceCloud_CL 
+        | where not(disabled)
+        | where event_type_s in~ (SalesforceEventType)
+        | extend LoginType_s = column_ifexists("login_type_s", "")
+        | extend LoginSubType_s = column_ifexists("login_sub_type_s", "")
+        | lookup EventResultLookup on login_status_s
+        | lookup EventTypeLookup on event_type_s
+        | lookup LogonMethodLookup on LoginType_s
+        | lookup LogonProtocolLookup on LoginSubType_s
+        | lookup TempEventResultLookup on request_status_s
+        | lookup DvcOsLookup on platform_type_s
+        | lookup UserTypeLookup on user_type_s
+        | project-rename
+            EventProductVersion = api_version_s,
+            EventOriginalResultDetails = login_status_s,
+            TargetUserId = user_id_s,
+            SrcIpAddr = source_ip_s,
+            EventOriginalUid = request_id_s,
+            TlsCipher = cipher_suite_s,
+            TlsVersion = tls_protocol_s,
+            HttpUserAgent= browser_type_s,
+            TargetUserScopeId = organization_id_s,
+            TargetUrl = uri_s,
+            TargetOriginalUserType = user_type_s,
+            ActorUsername = delegated_user_name_s,
+            ActorUserId = delegated_user_id_s,
+            TargetUsername = user_name_s
+        | extend
+            TimeGenerated = todatetime(tostring(split(timestamp_s, '.', 0)[0])),
+            EventVendor = 'Salesforce',
+            EventProduct='Salesforce Service Cloud',
+            EventCount = int(1),
+            EventSchema = 'Authentication',
+            EventSchemaVersion = '0.1.3',
+            TargetAppName = "Salesforce Dot Com(SFDC)",
+            EventUid = _ItemId,
+            EventOriginalType=event_type_s
+        | extend
+            TargetSessionId = coalesce(session_key_s, login_key_s),
+            SrcIpAddr = coalesce(SrcIpAddr, client_ip_s),
+            TargetUserScope = "Salesforce Organization",
+            TargetUserIdType = iff(isnotempty(TargetUserId), "SaleforceId", ""),
+            ActorUserIdType = iff(isnotempty(ActorUserId), "SaleforceId", ""),
+            TargetUsernameType = iff(isnotempty(TargetUsername), "UPN", ""),
+            ActorUsernameType = iff(isnotempty(ActorUsername), "UPN", ""),
+            User = coalesce(TargetUsername, TargetUserId),
+            Src = SrcIpAddr,
+            IpAddr = SrcIpAddr,
+            Dvc = EventProduct,
+            EventResult = coalesce(EventResult, TempEventResult),
+            Application = TargetAppName,
+            EventStartTime = TimeGenerated,
+            EventEndTime = TimeGenerated
+        | project-away
+            *_s,
+            *_t,
+            TenantId,
+            SourceSystem,
+            Computer,
+            MG,
+            ManagementGroupName,
+            Message,
+            RawData,
+            TempEventResult,
+            _ItemId,
```

Comment:
I think _ (underscore fields are hidden and not require to project away).

Reply:
@anki-narravula removing _resourceID, keeping _ItemId. _ItemId is visible if not done project-away